components/brave_wallet_ui/components/desktop/views/portfolio/components/nft-details/index.tsx

```diff
@@ -85,61 +87,80 @@ const NFTDetails = (props: Props) => {
 
   return (
     <StyledWrapper>
-      <NFTImage src={selectedAsset.logo} />
-      <DetailColumn>
-        <TokenName>
-          {selectedAsset.name} {
-            selectedAsset.tokenId
-              ? '#' + new Amount(selectedAsset.tokenId).toNumber()
-              : ''
+      {isLoading &&
+        <LoadingOverlay isLoading={isLoading}>
+          <LoadIcon />
+        </LoadingOverlay>
+      }
+      {nftMetadata &&
+        <>
+          <NTFImageIframe
+            onLoad={() => setIsImageLoaded(true)}
```

Comment:
we should add the `sandbox` attribute to this iframe: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox

Reply:
@diracdeltas I made this change